pkg/operator/ceph/file/filesystem.go

```diff
@@ -160,13 +156,31 @@ func newFS(name string, metadataPool *model.Pool, dataPools []*model.Pool, activ
 	}
 }
 
+// SetPoolSize method sets the sizes for MetadataPool and dataPool
+func SetPoolSize(context *clusterd.Context, clusterName, fsName string, dataPool []*model.Pool, metadataPool *model.Pool) error {
+	// generating the metadata pool's name
+	metadataPool = generateMetadataPoolName(fsName, metadataPool)
+	err := client.CreatePoolWithProfile(context, clusterName, *metadataPool, appName)
+	if err != nil {
+		errors.Wrapf(err, "failed to update metadata pool %q", *&metadataPool.Name)
```

Comment:
If there is an error you need to return it. Otherwise this line does nothing.

Reply:
I took the reference from here
https://github.com/rook/rook/blob/master/pkg/operator/ceph/pool/controller.go#L241
It will print the stack trace I believe.
What do you think?.

Reply:
you're missing `return`

Reply:
Fixed this too, thanks.